apis/core/v1alpha1/annotations.go

```diff
@@ -16,15 +16,14 @@ package v1alpha1
 const (
 	// AnnotationPrefix is the prefix for all ACK annotations
 	AnnotationPrefix = "services.k8s.aws/"
-	// AnnotationARN is an annotation whose value is an Amazon Resource Name,
-	// which is a globally-unique identifier, for the backend AWS service API
-	// resource. If this annotation is SET on a CR, that means the user is
+	// AnnotationAdopted is an annotation whose value is a boolean value,
+	// If this annotation is set to true on a CR, that means the user is
 	// indicating to the ACK service controller that it should expect a backend
 	// AWS service API resource to already exist (and that ACK should "adopt"
-	// the resource into its management). If this annotation is NOT SET on a
-	// CR, that means the user expects the ACK service controller to create the
-	// backend AWS service API resource.
-	AnnotationARN = AnnotationPrefix + "arn"
+	// the resource into its management). If this annotation is set to false on
+	// a CR, that means the user expects the ACK service controller to create
+	// the backend AWS service API resource.
+	AnnotationAdopted = AnnotationPrefix + "adopted"
```

Comment:
Been thinking more about this annotation and whether we really need it. I'm not so sure we do, at least in a boolean form. A couple questions that highlight my concerns:

1) If this annotation is not present on a CR, what does that mean?

2) Above, it states "If this annotation is set to false on a CR, that means the user expects the ACK service controller to  create the backend AWS service API resource." But this isn't really true, right? It's not the annotation's presence or lack of presence that dictates whether the service controller takes some action. Instead, it's the creation of an `AdoptedResource` CR explicitly by the Kubernetes user that is the signal for a service controller to grab the latest observed state of the referred-to resource instead of calling the API's Create operation.

3) Given 2), would it make more sense to add an annotation called `services.k8s.aws/adopted-on` that is a timestamp of when the `AdoptedResourceReconciler` successfully created the resource CR after pulling the latest observed state from the AWS API?

Reply:
Sure, I could see more value in this becoming a timestamp. It was initially a boolean so that you could easily run a filter of `services.k8s.aws/adopted=='true'` on all resource annotations, but I understand that timestamp probably provide that plus more.
This annotation isn't being used by any system, it really is just informational to the user.